lld/wasm/Writer.cpp

```diff
@@ -1467,15 +1469,29 @@ void Writer::createApplyDataRelocationsFunction() {
   {
     raw_string_ostream os(bodyContent);
     writeUleb128(os, 0, "num locals");
+    uint64_t off = os.tell();
     for (const OutputSegment *seg : segments)
       if (!config->sharedMemory || !seg->isTLS())
         for (const InputChunk *inSeg : seg->inputSegments)
           inSeg->generateRelocationCode(os);
```

Comment:
Would it be cleaner to return a boolean from `generateRelocationCode`? 

i.e.

```
wroteRelocations |= inSeg->generateRelocationCode(os);
...
if (!wroteRelocations) {
  return;
}
```

Not a big deal but it seems more explictly/obvious.

Reply:
done